moveit_commander/test/python_moveit_commander.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+import rospy
+import rostest
+import os
+
+from moveit_commander import RobotCommander
+
+
+class PythonMoveitCommanderTest(unittest.TestCase):
+    PLANNING_GROUP = "manipulator"
+
+    @classmethod
+    def setUpClass(self):
+        self.commander = RobotCommander("robot_description")
+        self.group = self.commander.get_group(self.PLANNING_GROUP)
+
+    @classmethod
+    def tearDown(self):
+        pass
+
+    def check_target_setting(self, expect, *args):
+        if len(args) == 0:
+            args = [expect]
+        self.group.set_joint_value_target(*args)
+        res = self.group.get_joint_value_target()
+        self.assertTrue(np.all(np.asarray(res) == np.asarray(expect)),
+                        "Setting failed for %s, values: %s" % (type(args[0]), res))
+
+    def test_target_setting(self):
+        n = self.group.get_variable_count()
+        self.check_target_setting([0.1] * n)
+        self.check_target_setting((0.2,) * n)
+        self.check_target_setting(np.zeros(n))
+        self.check_target_setting([0.3] * n, {name: 0.3 for name in self.group.get_active_joints()})
+        self.check_target_setting([0.5] + [0.3]*(n-1), "joint_1", 0.5)
+
+    def plan(self, target):
+        self.group.set_joint_value_target(target)
+        return self.group.plan()
+
+    def test_validation(self):
```

Comment:
rename something like `test__group__execute_path_validation` to make it easier to understand what this test does